test/integration_test/azure_disk_test.go

```diff
@@ -0,0 +1,123 @@
+//go:build integrationtest
+// +build integrationtest
+
+package integrationtest
+
+import (
```

Comment:
nit: group the import into 3 groups: golang internal, external github, portworx repo. see other test files for reference.